climada/hazard/trop_cyclone/trop_cyclone_windfields.py

```diff
@@ -100,7 +100,6 @@ def compute_angular_windspeeds(
     mask_centr_close: np.ndarray,
     model: int,
     model_kwargs: Optional[dict] = None,
-    cyclostrophic: bool = False,
```

Comment:
The proper way to deal with this is probably to mark the argument as deprecated rather than removing it right away

Reply:
Like this @emanuel-schmid ?